### PR TITLE
in-app direct capture

### PR DESCRIPTION
**Pull request by mcginty**

### CODE REVIEW

**Comment by McLoo** [comment]
What about moving the camera button above the emoji button as soon as you reach 3 lines of text? Would save some space.
 On my 4.3 inch display the compose field is already quite limited

**Comment by mcginty** [comment]
@McLoo was thinking about just hiding the camera button the same time that we switch to the "send" button instead of attach.

**Comment by McLoo** [comment]
@mcginty so much more reasonable :see_no_evil:

**Comment by mcginty** [comment]
updated, added the toggle animation

File: res/values/strings.xml [hunk 1]
```diff
@@ -512,7 +512,7 @@
     <string name="conversation_title_view__conversation_muted">Conversation muted</string>
 
     <!-- conversation_activity -->
-    <string name="conversation_activity__type_message_push">Send TextSecure message</string>
+    <string name="conversation_activity__type_message_push">Send via TextSecure</string>
```
Comment (moxie0) [inline res/values/strings.xml]: I kinda like the previous wording better

Comment (McLoo) [inline res/values/strings.xml]: Testing this PR. 
On my 4.3" display there is a line break after TextSecure, putting 'message' in the next line when in quick reply. The new phrase is in one line...

Comment (mcginty) [inline res/values/strings.xml]: Yep, I changed it to fit better on small screens. i also liked the previous wording better, it's just long.

Comment (2-4601) [inline res/values/strings.xml]: There will always be some languages that simply don't have a short translation for this.
Can the string be ellipsized so it doesn't break to a new line on small screens?

Comment (mcginty) [inline res/values/strings.xml]: I thought about that, but I was worried about a language that might show the equivalent of "Send via...", which is a security indication loss for us.

Comment (moxie0) [inline res/values/strings.xml]: Let's switch it back to Send TextSecure message and ellipsize it

Comment (2-4601) [inline res/values/strings.xml]: How about dropping "Send" altogether and just use "TextSecure message" and "Unsecured SMS"?

**Pull request closed without merging**